src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/IRazorLanguageQueryHandler.cs

```diff
@@ -1,8 +1,6 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
-#nullable disable
```

Comment:
Spotted that I missed these clarifications in the larger PR, included them here since the change is simple and this PR was tiny.